### PR TITLE
fix(TBD-13002): fix for hive wizard and universal

### PR DESCRIPTION
**Pull request by AlixMetivier**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution/resources/config/default/spark.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        "STANDALONE":{
+            "PORT":10000
+        },
+        "DATABASE":"default",
+        "HIVE_PRINCIPAL":"hive/_HOST@EXAMPLE.COM"
+    },
+    "HBASE":{
+        "PORT":2181
+    },
+    "HADOOP_CONFIG_LIST":["HDFS","MAPREDUCE2","YARN","WEBHCAT","HIVE","HBASE","OOZIE","SPARK","SQOOP","STORM","TEZ","ZOOKEEPER"]
```
Comment (sponomarova) [inline main/plugins/org.talend.hadoop.distribution/resources/config/default/spark.json:30]: just wondering if "STORM" should be here?

Comment (sponomarova) [reply main/plugins/org.talend.hadoop.distribution/resources/config/default/spark.json:30]: it is about to be removed. No?

Comment (AlixMetivier) [reply main/plugins/org.talend.hadoop.distribution/resources/config/default/spark.json]: I can remove it :)

**Review by lbourgeois — APPROVED** [review]
LGTM

**Review by sponomarova — APPROVED** [review]
lgtm

**Pull request merged**